kilometer-backend-domain/src/main/java/com/kilometer/domain/archive/like/LikeService.java

```diff
@@ -20,31 +20,31 @@ public LikeDto findByArchiveIdAndUserId(Long archiveId, Long userId) {
         Preconditions.checkNotNull(archiveId, "Archive id must not be null");
         Preconditions.checkNotNull(userId, "User id must not be null");
 
-        Archive likedArchive = Archive.builder().id(archiveId).build();
+        ArchiveEntity likedArchiveEntity = ArchiveEntity.builder().id(archiveId).build();
```

Comment:
이렇게 한 줄에 메서드 호출을 하나씩만 가져가는 방식은 어떤가요? 메서드 체이닝이 한 줄에서 발생하면 가독성이 떨어지는 것 같아요😭.
```suggestion
ArchiveEntity likedArchiveEntity = ArchiveEntity.builder()
                .id(archiveId)
                .build();
```

Reply:
이 부분은 `Archive` 엔티티 객체를 `ArchiveEntity`로 변경하면서 이름만 변경했던 것 같아요! 
추가해서 다시 올리도록하겠습니다!